pod/config.go

```diff
@@ -44,6 +44,7 @@ type Config struct {
 	LogS3WriterIAMRole       *string
 	LogS3BucketName          *string
 	LogS3PathPrefix          *string
+	NetworkMode              *uint32
```

Comment:
Why wouldn't we store the string rather than the proto number in the annotation?

Reply:
Oh. Hmmm. Yea.....